README.md

```diff
@@ -10,6 +10,16 @@ To see information regarding forcing dataset development for the Coupled Model I
 
 To register your institution or source/dataset, please submit an issue/ticket following the instructions on the [input4MIPs_CVs issue page](https://github.com/PCMDI/input4MIPs_CVs/issues/new). Follow the same procedure to request a change in any other CV.
 
+### Registering a new input4MIPs forcing dataset
+
+[ZN to write, rough steps, current guess below]
```

Comment:
@durack1 not sure if you have other thoughts on whether this information should live here or somewhere else, but obviously we need to capture the 'how do I get my data onto ESGF' process somewhere